Bottom: dd1764753e2a8d59fa9599f28a227fc1b1586117
Top:    b5b0018b79bc03e6a1efc99625c16c9897813165
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2022-12-28 13:45:59 +0100

bsp initial langdale support

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index 299a0f74bb..c203771e12 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus dunfell gatesgarth"
+LAYERSERIES_COMPAT_{{=machine}} = "langdale"
